composer.json

```diff
@@ -1,15 +1,24 @@
 {
-    "name": "lycheeorg/lychee-laravel",
+    "name": "lycheeorg/lychee",
     "description": "A great looking and easy-to-use photo-management-system you can run on your server, to manage and share photos.",
     "homepage": "https://lycheeorg.github.io/",
+    "readme": "readme.md",
+    "support": {
+        "source": "https://github.com/LycheeOrg/Lychee",
+        "issues": "https://github.com/LycheeOrg/Lychee/issues",
+        "docs": "https://lycheeorg.github.io/docs/",
+        "chat": "https://gitter.im/LycheeOrg/Lobby"
+    },
     "keywords": [
         "framework",
         "laravel"
     ],
```

Comment:
If we are on it, I wonder whether these keywords make any sense. The seem to have been copied from a Laravel application template, but Lychee is not a framework. I would have expected keywords like

 - `"photo management"`,
 - `"gallery"`,
 - `"album software"`,
 - `"image organizer"`,
 - etc.

The latter came to my mind, because Wikipedia calls it like this: https://en.wikipedia.org/wiki/Image_organizer, https://en.wikipedia.org/wiki/Comparison_of_photo_gallery_software